neo/sys/posix/posix_main.cpp

```diff
@@ -688,11 +688,7 @@ Posix_Cwd
 const char* Posix_Cwd()
 {
 	static char cwd[MAX_OSPATH];
```

Comment:
I just noticed Doom3BFG's `MAX_OSPATH` is 256, which is bullshit of course, it should be set to `PATH_MAX` or maybe (like classic doom3 or at least dhewm3 does) FILENAME_MAX (which is both 4096 on Linux anyway).